presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionModule.java

```diff
@@ -63,7 +63,7 @@ public void configure(Binder binder)
         httpClientBinder(binder)
                 .bindHttpClient("nativeExecution", ForNativeExecutionTask.class)
                 .withConfigDefaults(config -> {
-                    config.setRequestTimeout(new Duration(10, SECONDS));
+                    config.setRequestTimeout(new Duration(1, MINUTES));
```

Comment:
Would it be possible to make this configurable, so that we can keep 10s default timeout, but change it to 60s or something else in production?

CC: @majetideepak

Reply:
Agree with making this configurable. Seems doable by using the `connectorConfig`
Probably can add a config for `MaxConnectionsPerServer` also.

Reply:
Turns out here is just setting the default value, we can override the default by setting the `nativeExecution.http-client.request-timeout` (similar to other presto http client). Will close this PR.

Reply:
@mbasmanova Is this config for Spark only? I notice timeout for httpClient in `presto_cpp` is hardcoded as 10s like https://github.com/prestodb/presto/blob/master/presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp#L74

While presto java control it via `exchange.http-client.request-timeout`